modules/@apostrophecms/layout-widget/ui/apos/components/AposAreaLayoutEditor.vue

```diff
@@ -35,6 +35,7 @@
       <template v-else>
         <AposAreaMenu
           :context-menu-options="contextMenuOptions"
+          :field-id="fieldId"
```

Comment:
A new shiny bug (Vue dev warning about missing required prop). Most probably due to changes from `main`. It's good now.